app/user-quickfiles/controller.js

```diff
@@ -24,4 +25,6 @@ export default Controller.extend(Analytics, {
             }
         },
     },
+
+    jiraComponent: config.microfeedback.JIRA.components.QuickFiles,
```

Comment:
jiraComponent is very environment-specific. Maybe something like `microfeedbackOptions`